Tests/SonarQube.MSBuild.Tasks.IntegrationTests/TargetsTests/RoslynTargetsTests.cs

```diff
@@ -18,6 +19,8 @@ public class RoslynTargetsTests
     {
         private const string RoslynAnalysisResultsSettingName = "sonar.cs.roslyn.reportFilePath";
 
+        private static readonly string[] SonarLintAnalyzerFileNames = new string[] { "SonarLint.dll", "SonarLint.CSharp.dll" };
```

Comment:
The changes to the tests are to handle the presence of multiple SonarLint assemblies.